packages/cloudflare/src/cli/templates/worker.ts

```diff
@@ -54,7 +54,7 @@ export default {
 			// @ts-expect-error: resolved by wrangler build
 			const { handler } = await import("./server-functions/default/handler.mjs");
 
-			return handler(reqOrResp, env, ctx);
+			return handler(reqOrResp, env, ctx, request.signal);
```

Comment:
@sommeeeer what do you think about making this PR a minor change to signal users that there is a change in the worker template?